Cookbook/Technical-Documents/DebugBuilder.md

```diff
@@ -0,0 +1,96 @@
+Debug Builder
+======================
+
+[\[CNSMR\-1602\] Add \`DebugAuthenticatedRootBuilder\` for faster UI debugging iteration by inamiy · Pull Request \#7645 · Babylonpartners/babylon\-ios](https://github.com/Babylonpartners/babylon-ios/pull/7645)
+
+Above PR introduces `DebugAuthenticatedRootBuilder` that allows to directly show a target view builder after authentication.
+
+This feature improves overall UI debugging iteration speed.
+
+## How to make a debug-builder
+
+1. Add a new case in `enum DebugBuilderPattern`
+2. Create a (mock) builder in `DebugAuthenticatedRootBuilder`
+3. Update `Settings.bundle/Root.plist` by adding a new case key
+4. Set `Settings.bundle`'s userDefaults flag via `Settings.app`
+5. Launch the app
+
+## Example
+
+`LocalFeatureSwitches.swift`
+
+```diff
+ public enum DebugBuilderPattern: String {
+     case none
```

Comment:
Update to the new name chosen once https://github.com/Babylonpartners/babylon-ios/pull/7645/files#r293449642 has been adressed

Reply:
```suggestion
     case disabled
```